### ISSUE TITLE
Instancelist output

### ISSUE THREAD

**Issue by mckraken** [mdfilio/go-aws-utils#2]
The output doesn't line up with longer instance ids (some items edited for obfuscation):

```
Instance ID      State        OS         Type               VPC          Subnet        Public IP        Private IP       Name
----------------------------------------------------------------------------------------------------------------------------------------
i-06dd7ee0728d05a0b running      Linux     t2.medium       vpc-fa00009e  subnet-5c00002a   52.210.000.000   10.244.000.00    ECS-instance 
```

**Comment by mdfilio** [comment]
I was aware of this.. but was lazy and haven't updated.

**Comment by mdfilio** [comment]
fixed in commit bf31af2
